ActindoWorkFlow/UserInterface/08_CoreActions.md

```diff
@@ -0,0 +1,247 @@
+Other workflow actions are available depending on the plugins installed in the current system.
+In the following, the core actions, their use and their settings are described in detail:
+
+- [Change process priority](#change-process-priority)
+- [Multiply input action](#multiply-input-action)
+- [Execute PHP code](#execute-php-code)
+- [Split by criterion](#split-by-criterion)
+- [Start subprocess](#start-subprocess)
+- [Switch case action](#switch-case-action)
+- [Update process name and search string](#update-process-name-and-search-string)
+- [Wait for criterion](#wait-for-criterion)
+- [Wait for parallel input](#wait-for-parallel-input)
+- [Waiting action](#waiting-action)
+
+
+
+## Change process priority
+
+![Change process priority](../../Assets/Screenshots/ActindoWorkFlow/Workflows/CoreActions/ChangeProcessPriority.png "[Change process priority]")
+
+The *Change process priority* core action is used to change the preconfigured priority of the process while the process is already running.   
+The data runs via the *loop_through* input port into the workflow action and is output via the *p* output port. However, the data is only output when data has also been incoming via the *priority* input port. The input value of the *priority* input port must be an integer. This integer will be used as the new priority.
+
+[comment]: <> (ticket in arbeit: static inputs sollen im priority input port möglich sein, ändern sobald möglich)
+
+### Configuration   
+
+This core action has no further configuration settings.
+
+
+
+## Multiply input action
+
+![Multiply input action](../../Assets/Screenshots/ActindoWorkFlow/Workflows/CoreActions/MultiplyInputAction.png "[Multiply input action]")
+
+The *Multiply input action* core action is used to output the data coming in via one input port to two output ports. This core action is often used when the same data is needed for two different purposes. To merge the duplicated data again, the *Wait for parallel input* core action can be used, see [Wait for parallel input](#wait-for-parallel-input) .   
+The data runs via the *p* input port into the workflow action and is output via both the *p0* and the *p1* output port.
+
+[comment]: <> (ticket in arbeit: mehr als zwei output port sollen hinzukommen)
+
+### Configuration   
+
+This core action has no further configuration settings.
+
+
+
+## Execute PHP code
+
+![Execute PHP code](../../Assets/Screenshots/ActindoWorkFlow/Workflows/CoreActions/ExecutePHPCode.png "[Execute PHP code]")
+
+The *Execute PHP code* core action is used to execute a custom PHP code defined in the configuration.
+As the complete functionality of the PHP code is accessible, this core action enables complete variability in programming and is therefore often used when more complex actions which require more logic need be executed within a workflow step.   
```

Comment:
```suggestion
As the complete functionality of the PHP code is accessible, this core action enables complete variability in programming. Therefore, it is often used when more complex actions which require more logic need be executed within a workflow step.   
```
Alternativ: Therefore, it is often used when more complex actions, which require more logic/requiring more logic, need be executed within a workflow step.